website/docs/cloud-docs/policy-enforcement/prewritten-library.mdx

```diff
@@ -0,0 +1,22 @@
+---
+page_title: Pre-written policy library
+description: HashiCorp authors and maintains a library of pre-written Sentinel policies that enforce CIS and other compliance standards. Learn about the available pre-written policies. 
+---
+
+# Pre-written policy library reference
+
+This topic provides reference information about the Sentinel policy libraries that HashiCorp authors and maintains. For instructions on how to run the policy libraries, refer to [Run pre-written Sentinel policies ](/terraform/cloud-docs/policy-enforcement/prewritten-sentinel).
+
+## Center for Internet Security (CIS)
+
+The Center for Internet Security (CIS) is a non-profit organization that publishes standards for configuring secure cloud services. Refer to the [CIS website](https://www.cisecurity.org) for additional information.  
+
+HashiCorp publishes pre-written policies that support the following CIS benchmarks.
+
+### AWS
+
+- Amazon Web Services Foundations version 1.2
+- Amazon Web Services Foundations version 1.4 
+- Amazon Web Services Foundations version 3.0
+
```

Comment:
we usually have a sentence after headers before we jump directly into bullet points

also, do we have different policy libraries for each one of the versions? can we link out directly to each one?

Reply:
I was leaning on this guidance w/r/t to going straight to a list:

"You can omit the introductory sentence when the list immediately follows a heading, such as the list of requirements on a usage page."  This is not a usage page but it seems to fit under this statement. 

The change for GA is that there is now a single repo for a for all of them organized by resource type. I wrote it out this way because in my experience, people look usually need to comply with a version of standard.